addons/stock_picking_wave/controllers/main.py

```diff
@@ -1,16 +1,17 @@
-from openerp.addons.web import http
-from openerp.addons.web.http import request
+# -*- coding: utf-8 -*-
+# Part of Odoo. See LICENSE file for full copyright and licensing details.
 
+from openerp import http
+from openerp.http import request
 
-class picking_wave_report(http.Controller):
-    @http.route('/report/stock_picking_wave.report_pickingwave/<ids>', type='http', auth='user', 
-                website=True)
-    def report_picking_wave(self, ids):
-        self.cr, self.uid, self.pool = request.cr, request.uid, request.registry
+
+class PickingWaveReport(http.Controller):
```

Comment:
Well, move the removal commit before migrating, so that we don't have to review the migration of something we want to remove ;) .